core/domain/param_domain.py

```diff
@@ -57,25 +87,49 @@ class ParamChange(object):
     """Value object for a parameter change."""
 
     def __init__(self, name, generator_id, customization_args):
-
+        """Initialze a ParamChange object with the specified arguments
+
+        Args:
+            name: unicode. The name of the parameter.
+            generator_id: unicode. The type of generator used to create the
+            parameter. eg. Copier
+            customization_args: dict. A dict containing all the arguments
+            including the value of the parameter.
+        """
         # TODO(sll): Check that all required args for customization exist in
         # customization_args.
-
         self._name = name
         self._generator_id = generator_id
         self._customization_args = customization_args
 
+
     @property
     def name(self):
+        """The name of the changing parameter.
+
+        Returns:
+            self.name: unicode. The name of the parameter.
+        """
         return self._name
 
     @property
     def generator(self):
+        """The generator of the changing parameter
```

Comment:
I don't understand what "the generator of the changing parameter" means. Perhaps:

    The value generator used to define the new value of the changing parameter.